SX1272/SX1272_LoRaRadio.cpp

```diff
@@ -1591,7 +1594,6 @@ void SX1272_LoRaRadio::setup_interrupts()
  */
 void SX1272_LoRaRadio::set_low_power_mode(bool status)
 {
-
     if( RadioIsActive != status )
```

Comment:
Could you please change the style here to Mbed-OS style.

Reply:
Let's leave style changes to a another commit.